### PR TITLE
Works and About section for mobile

### PR DESCRIPTION
**Pull request by awaisanwar544**
Completed the works and about section.

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -32,9 +32,204 @@ <h1 class="center-text primary-text-color main-heading">Hey there. I&apos;m Mari
...
+                    <div class="lang-item">
+                        <p>css</p>
+                    </div>
+                    <div class="lang-item">
+                        <p>Javascript</p>
+                    </div>
+                    <div class="lang-item">
+                        <p>html</p>
+                    </div>
+                </div>
```
Comment (Cathella) [inline app/index.html]: Hi @awaisanwar544, I am impressed by the great effort you have invested in this project. I looks awesome and near ⭐ perfect ⭐. However we got a few changes to make right here.

- [ ] In reference to the project requirement below 👇🏻. I am suggesting to wrap the list of project technologies in a `<ul>` tag.

image

File: app/index.html [hunk 2]
```diff
@@ -32,9 +32,204 @@ <h1 class="center-text primary-text-color main-heading">Hey there. I&apos;m Mari
...
+                    <button class="button work-item-button primary-font">See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section>
+        <section id="about" class="content">
+            <img class="about-bg-1" src="./assets/i_about_me_t.png" alt="">
+            <img class="about-bg-2" src="./assets/i_about_me_b.png" alt="">
```
Comment (Cathella) [inline app/index.html]: - [ ] I think these images are meant to be proper background image (decorations are always background images, not img tags). What do you think 🤔 ?

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🚩 Status: Changes Required 🛑

### Hi @awaisanwar544 ✋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## The Good points 💪🏻 

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ The sections use the font and colors defined for the template
✔️ Used Flexbox to place elements in the page.
✔️ Used CSS Grid to place elements in the page.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/index.html (hunk 1 above)
Comment (awaisanwar544) [reply app/index.html]: changes made: wrapped in `<ul>` instead of `<div>`

File: app/index.html (hunk 2 above)
Comment (awaisanwar544) [reply app/index.html]: [changes done] decorations are now background images

**Review by ReshyResh — APPROVED** [review]
# Hi @awaisanwar544 , 

### Great job implementing the changes required by the other reviewer!
#### Your project is approved, you can now merge it! 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**